.github/actions/setup-build-env/action.yml

```diff
@@ -1,26 +1,32 @@
 name: "Setup build environment"
 description: "Setup the runner with all the tools we need to build our project."
 inputs:
-    mock-google-services:
-        description: "Mock the google-services.json file inside the gradle modules."
-        default: "false"
-        required: false
+  mock-google-services:
+    description: "Mock the google-services.json file inside the gradle modules."
+    default: "false"
+    required: false
+  cache-disabled:
+    description: When 'true', all caching is disabled. No entries will be written to or read from the cache.
+    required: false
+    default: "false"
```

Comment:
In order to be able to debug cache issues I introduced this flag, but its usage should not land on `master`.